spec/support/request_spec_helper.rb

```diff
@@ -22,4 +23,12 @@ def options(*args)
       copy_session_variables!
     end
   end
+
+  def mock_api_doorkeeper_pass
```

Comment:
looks like this is an exact dupe of the method in `spec/support/controller_spec_helper.rb` -- possible to share logic?